src/controllers/UsersController.php

```diff
@@ -386,6 +386,42 @@ public function deleteUser($request, $db)
         $view->setResponseCode(204);
     }
 
+    /**
+     * Allow users to be set as trusted
+     *
+     * @param $request
+     * @param $db
+     *
+     * @return null
+     *
+     * @throws Exception
+     *
+     */
+    public function setTrusted($request, $db)
+    {
+        if (false == ($request->getUserId())) {
+            throw new Exception("You must be logged in to change a user account", 401);
+        }
+
+        $user_mapper = $this->getUserMapper($db, $request);
+        if (!$user_mapper->isSiteAdmin($request->getUserId())) {
+            throw new Exception("You must be an admin to change a user's trusted state", 401);
+        }
+
+        $userId = $this->getItemId($request);
+        if (!is_bool($trustedStatus = $request->getParameter("trusted", null))) {
+            throw new Exception("You must provide a trusted state", 400);
+        }
+
+        if (!$user_mapper->setTrustedStatus($trustedStatus, $userId)) {
+            throw new Exception("Unable to update status");
```

Comment:
I should be returning a code, however it should probably be in the 5xx range - this error will happen if the DB has refused to update the field for some reason, so that's going to be a server error rather than a user error - I'm thinking a nice, vague 500 :p